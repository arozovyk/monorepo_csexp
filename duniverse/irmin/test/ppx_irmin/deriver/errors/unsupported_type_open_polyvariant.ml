type t = [> `On | `Off ] [@@deriving irmin]
