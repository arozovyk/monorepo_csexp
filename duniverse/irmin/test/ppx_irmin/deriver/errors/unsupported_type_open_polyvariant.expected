File "unsupported_type_open_polyvariant.ml", line 1, characters 0-43:
Error: ppx_irmin: open polymorphic variant [> `On  | `Off ] encountered. Polymorphic variants must be closed.
